public/assets/ui/trail-spine.ts

```
@@ -0,0 +1,88 @@
+import { getSession } from '../workflow-trail';
+
+interface SpineStep {
+  id: number;
+  tool: string;
+  status: 'ok' | 'err' | 'denied';
+  ms: number;
+  summary: string;
+  expanded: boolean;
+}
+
+let expanded = new Set<number>();
```

ESLint

[error] 'expanded' is never reassigned. Use 'const' instead.

(prefer-const)